src/components/OperatorActivations/OperatorActivations.tsx

```diff
@@ -40,7 +40,7 @@ const OperatorActivations = () => {
   const [params, setParams] = useState<GetOrgsParams>();
   const {
     data: operators,
-    isLoading,
+    isPending: isLoading,
```

Comment:
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
    isPending,
```